src/service/fixed-layer.js

```diff
@@ -162,6 +234,19 @@ export class FixedLayer {
           ' as Google\'s because the fixed or sticky positioning might have' +
           ' slightly different layout.');
     }
+  }
+
+  /**
+   * @param {!Node} node
+   * @param {boolean=} opt_lightboxMode
+   * @private
+   */
+  scanNode_(node, opt_lightboxMode) {
+    this.trySetupSelectorsNoInline(node, devAssert(this.fixedSelectors_),
```

Comment:
We could just instantiate `this.fixedSelectors_` in the constructor.

Reply:
Done.